anoncreds/src/ffi/cred_def.rs

```diff
@@ -56,6 +58,29 @@ pub extern "C" fn anoncreds_create_credential_definition(
     })
 }
 
+#[no_mangle]
+pub extern "C" fn anoncreds_credential_definition_get_attribute(
+    handle: ObjectHandle,
+    name: FfiStr,
+    result_p: *mut *const c_char,
+) -> ErrorCode {
+    catch_error(|| {
+        check_useful_c_ptr!(result_p);
+        let cred_def = handle.load()?;
+        let cred_def = cred_def.cast_ref::<CredentialDefinition>()?;
+        let val = match name.as_opt_str().unwrap_or_default() {
+            "schema_id" => match cred_def {
+                CredentialDefinition::CredentialDefinitionV1(cred_def) => {
+                    cred_def.schema_id.to_string()
+                }
+            },
+            s => return Err(err_msg!("Unsupported attribute: {}", s)),
+        };
+        unsafe { *result_p = rust_string_to_c(val) };
+        Ok(())
+    })
+}
```

Comment:
This method was removed by me, but added it back. It can get the schema property from a handle, but I think we can remove it as it does not add any value.

Reply:
It's used by the python wrapper, and avoids the caller having to unwrap different versions of a credential definition to find the property (although at the moment there's only one version). There's a similar method for revocation registry definitions that supports more fields: https://github.com/hyperledger/anoncreds-rs/blob/38e9a55c379694112b04c9def4ad3807e79bbe4c/anoncreds/src/ffi/revocation.rs#L172

Reply:
Incidentally I'm not sure there's any benefit to supporting multiple 'versions', it just complicates the code everywhere, although I wouldn't rip it out without some consideration.

Reply:
I removed the V1 versions in another PR.